client/analytics/report/products/table.js

```diff
@@ -160,6 +160,30 @@ class ProductsReportTable extends Component {
 		} );
 	}
 
+	getSummary( totals ) {
+		if ( ! totals ) {
+			return [];
+		}
+		return [
+			{
+				label: _n( 'product sold', 'products sold', totals.products_count, 'wc-admin' ),
+				value: totals.products_count,
```

Comment:
Should all of the numbers use [`numberFormat()`](https://github.com/woocommerce/wc-admin/blob/master/client/lib/number/index.js#L10) here?

Reply:
Absolutely! Actually, I saw many other tables are missing `numberFormat()` in the summary and rows too, so I will create another PR with fixes for the other tables.